user_sync/app.py

```diff
@@ -353,6 +359,17 @@ def begin_work(config_loader):
     directory_groups = config_loader.get_directory_groups()
     rule_config = config_loader.get_rule_options()
 
+    ssl_cert_verify = rule_config.get('ssl_cert_verify', True)
+    user_sync.connector.umapi.UmapiConnector.ssl_cert_verify = ssl_cert_verify
+    user_sync.post_sync.connectors.sign_sync.SignClient.ssl_cert_verify = ssl_cert_verify
```

Comment:
See my previous note - modifying a class property can have unexpected and often undesirable side effects.